### DISCUSSION TITLE
Убрать уведомления.

### DISCUSSION THREAD

**Discussion by lelicopter** [Storik4pro/goodbyeDPI-UI#43] [Ideas]
Свернул, обнова вышла, развернул, обновил, снова свернул - 5 уведомлений, зачем?? куда их, солить?! =))
ИМХО, достаточно только уведомления о наличии обновлении.

зы: да и они криво работают - "обновились какие-кто компоненты". а обновляешь всё подряд - говорит нет обновлений нигде =)

**Comment by Storik4pro** [comment]
Вы можете управлять уведомлениями на вкладке "система"->"уведомления"
